sbol_utilities/graph_sbol.py

```diff
@@ -4,7 +4,7 @@
 import argparse
 
 
-def graph_sbol(doc, outfile='out'):
+def graph_sbol(doc: sbol3.Document, file_format: str, view_now: bool, outfile: str ='out'):
```

Comment:
I believe that file_format and view_now should have defaults.
I'd suggest view_now defaults to false (matching the -v to enable below) and format defaults to PDF

Reply:
Alright